package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -22,89 +22,180 @@ define([
   "use strict";
 
   /**
-   * @name View
+   * @name pentaho.visual.base.View
    * @memberOf pentaho.visual.base
    * @class
    * @extends pentaho.lang.Base
    * @abstract
    * @classDesc This is the base class for visualizations.
    *
+   * A container is expected to instantiate a `View` with a reference to a `Model` instance,
+   * which may not be immediately valid.
+   *
+   * Over time, the container mutates the `Model` instance and triggers events.
+   * In response, the method [_onChange]{@link pentaho.visual.base.View#_onChange} is
+   * invoked to process the events, which may cause the `View` to update itself.
+   *
    * @description Creates a base `View` instance.
```

Comment:
This description is strange as the class is abstract. Maybe "Initializes a `View` instance." ?